instrumentation/jaxrs-client/jaxrs-client-2.0/src/main/java/io/opentelemetry/auto/instrumentation/jaxrsclient/v2_0/JaxRsClientTracer.java

```diff
@@ -54,4 +50,14 @@ protected String requestHeader(ClientRequestContext clientRequestContext, String
   protected String responseHeader(ClientResponseContext clientResponseContext, String name) {
     return clientResponseContext.getHeaderString(name);
   }
+
+  @Override
+  protected Setter<ClientRequestContext> getSetter() {
+    return null;
```

Comment:
There `io.opentelemetry.auto.instrumentation.jaxrsclient.v2_0.InjectAdapter.SETTER`, why do you return `null`?

Reply:
HttpTextFormat.Setter<MultivaluedMap>.  JaxRsClientTracer has 'ClientRequestContext' request type.

Reply:
As in case of version 2 below, you can change SETTER

Reply:
I did try. it doesn't support clientrequestcontext.

Reply:
by the way return null in many other places too.  @anuraaga insisted to make this method abstract that's why it returns null for tracers that don't apply to the setter type.

Reply:
i think you can do the same thing here that you did for v1.1: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/922/commits/6702d6a729d94bde3d6e7daa78d5ce844c0bb5a5